processes/vuln_db.md

```diff
@@ -72,8 +72,7 @@ Time window for introducing changes:
 - Changes that affect existing database fields will provide a 30 days time-frame through announcement for any party to chime in and weigh on the changes proposed
 - Changes that extend the database fields with newly proposed fields will be added as the proposed Pull-Request is merged
 
-Note: we recommended to subscribe to the repository's issue queue in order to receive updates in a timely manner for both new issues that may influence database changes, as well as new vulnerabilities pushed to the database or changes made to existing.
-
+Note: we recommended to subscribe to the repository's issue queue in order to receive updates in a timely manner for both new issues that may influence database changes, as well as new vulnerabilities pushed to the database or changes made to existing. Also, all Pull-Request that add a new vulnerability are labled with `vulnerabilities` which makes them easily filtered and searchable.
```

Comment:
issue queue -> issue tracker (I know that's not really part of this change)
Pull-Request -> pull request